packages/sdk-browsersigner/index.ts

```diff
@@ -0,0 +1,104 @@
+import { CredentialSigner, KeyAssertion, UserRegistrationChallenge, KeyAttestation, AllowCredential } from '@dfns/sdk'
+import { toBase64Url, toHex, exportPublicKeyInPemFormatBrowser, rawSignatureToAns1, generateRandom } from '@dfns/sdk/utils'
+import { Buffer } from 'buffer'
+
+export class BrowserKeySigner implements CredentialSigner<KeyAssertion> {
+  constructor(
+    private options: {
+      keyPair: CryptoKeyPair
+      credId?: string
+      appOrigin: string
+      crossOrigin?: boolean
```

Comment:
maybe we can ditch those fields ? Since
- `origin` should not be needed anymore for a Key Cred: we have a pending [PR](https://github.com/dfnsco/dfns/pull/1269) for that
- `crossOrigin`: not needed as well, doesn't make sense for key creds. Cody only added that field to be "ISO webauthN", I believe that field is added in webauthN `clientData` in assertions. Afaiu, webauthN sets it to `true` if the cred was used in a document that's a child of another site (iframe/popup), which has another origin ([in the spec](https://www.w3.org/TR/webauthn-3/#dictionary-client-data) there's also another field `topOrigin` that can be added to clientData). But I don't think we actually verify this [during assertion validation](https://github.com/dfnsco/dfns/blob/8dd658c1ab3d19d217db92928b101dc31f2b06f2/services/auth/dockers/security-auth-management/src/utils/webauthn.ts#L179).

Reply:
I removed `crossOrigin`. I'll create a separate PR to remove origin once we deploy the corresponding backend PR (it's easier for testing to keep it for now).